W&B Training: Serverless RL launch (#1682)
wandb/docs: content/en/guides/training/serverless-rl/available-models.md

---
title: "Available Models"
linkTitle: "Available Models"
weight: 40
description: >
  See the models you can train with Serverless RL.
---

> Serverless RL currently only supports a single open-source foundation model for training. We will add more models soon.

Reviewer: Can we avoid or minimize the future-facing language?

Author: Deleted!

> If there's a particular model you're interested in getting support for, email use at support@wandb.com.

Reviewer: ```suggestion
To express interest in a particular model, contact [support](mailto:support@wandb.ai).
```
